wasp/apps/clock.py

```diff
@@ -96,14 +97,16 @@ def _draw(self, redraw=False):
         # Format the month as text
         month = now[1] - 1
         month = MONTH[month*3:(month+1)*3]
+        day = now[7] - 7
```

Comment:
now[7] is not the day of the week (0..6), it is the day of the year (0..364 except on leap years). In other words this code will break on Thursday!

now[6] is the field that gives the day of the week.

Reply:
Now 6 yes you are right.... but still brake